readme/template-guide-readme.md

```diff
@@ -0,0 +1,101 @@
+This section is optional. 
+
+A project logo can be helpful for users to visually identify your project. 
+
+Embed relevant badges to your README to add credibility to your project. Badges highlight important information and can help users determine project fit quickly. [Learn more about badges.](https://github.com/badges/shields)
+
+### About the "Project name" section
+The project name should be prominent in the README and easy for users to identify. You may want to consider adding the following to help identify the project:
+- Project URL
+- Name(s) of project owner(s)
+
+*Note: Consider the stage your project is at and how much information is required in your README. Names associated with the project may be better located in a separate contact document.*
+
+
+### About the "Table of contents" section
+This section is optional. 
+
+A table of contents can be useful in the case of lengthy READMEs. As the project evolves, the README may require a table of contents for information organization.
+
+*Note: For Markdown files, GitHub now automatically generates a table of contents in the file Header. [Learn more about the GitHub TOC here.](https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/)*
+
+### About the "Project description" section
+The project description is the most critical part of your README. Describe what the project does, why the reader should care about your project, and how it can help the reader. This section should be succinct, use strong verbs, and help the reader determine whether the project is of use to them.  
+
+Screenshots or videos may also be included in this section to help readers understand your project. Any assets you direct users to should exist in the repo.
+
+You can use the below project description format to start filling in your section. An example is provided.
+
+
+With *(this project)* you can *(verb)* *(noun)*...
+
+*(This project)* helps you *(verb)* *(noun)*....
+
+Unlike *(alternative)*, *)this project)* *(verb)* *(noun)*... 
+
+
+---
+
+Example: With the Chronologue project, you can discover astronomical events throughout documented history. This project helps you understand the evolution of space in a streamlined format. Unlike googling and searching for important astronomical events, the Chronologue project outlines historical events with an easy to use and simple interface.
+
+---
+
+### About the "Who is the project for" section
+Identify your project audience and state who can use the project and under what terms. 
+
+### About the "Project dependencies" section
+List any pre-requisites a user needs to interact with your project. Include relevant links to installation instructions or resources. 
+
+### About the "Instructions for using project" section
+Describe in clear steps how to get, install, configure, run, and troubleshoot the project. Use a table for troubleshooting common issues to clearly identify issues and solutions.
+ 
```

Comment:
**Question (blocking, missing content):** I can't quite tell, but it looks like there's a small discrepancy between the "About the..." headings here and the template. Like here you use "Instructions for using project" but that's kind of different from the headings I see in the actual template. In my mind, they should be direct mirrors of each other, so I'd recommend building out that content a little more here. Your thoughts?